### ISSUE TITLE
Add browsers changes

### ISSUE THREAD

**Issue by ai** [c2corg/browserslist-update-action#226]
Thanks for amazing project. I added it to [Browserslist docs](https://github.com/browserslist/browserslist/#readme).

Right now, `browserslist --update-db` prints how DB update changed target browser (to explain why it is important to update DB).

Am I right that PR text is static? Can we add `diff` code section with the data from `browserslist --update-db`?

**Comment by brunobesson** [comment]
Hi, thanks for your feedback.

Indeed the PR text is kind of static, although you can override the default title and body with `title` and `body` action inputs.

I am not sure to understand what you woud like to see in the PR text. Something like the following?

```text
Caniuse database has been updated from version 1.0.30001355 (installed) to 1.0.30001357 (latest). Review changes, merge this PR and have a 🍺.
```

**Comment by ai** [comment]
Nope, the idea is to show off how target browsers will be changed on db update.

Right now CLI tool prints this diff to terminal. We can use it.

Below will be an example of the text I suggest (I need GitHub formatting in example sou will use separated comment to show the example):

**Comment by ai** [comment]
Caniuse database has been updated from version 1.0.30001355 (installed) to 1.0.30001357 (latest).

Target browsers changes:

```diff
- chrome 95
- firefox 99
+ chomre 101
+ firefox 100

**Comment by brunobesson** [comment]
Ok, I get it.
I'll try to make an update pretty soon, might be only a few days, or a week or too. Do not hesitate to propose an update for this if you feel so, or to ping me again if you see othing coming 🙂

**Comment by brunobesson** [comment]
Fixed in v1.3.1 (v1). I welcome your feedback.
